legal-api/src/legal_api/services/business_details_version.py

```diff
@@ -812,6 +815,9 @@ def resolution_json(resolution_revision) -> dict:
     @staticmethod
     def business_revision_json(business_revision, business_json):
         """Return the business revision as a json object."""
+        if not business_revision:
+            return business_json
```

Comment:
This is not required.  I believe you should also have a non-null value for `business_revision`.

You should be updating this [function](https://github.com/bcgov/lear/blob/1b40e6d07ba8b1d090b4ffd21ca3aaef102f1533/legal-api/src/legal_api/services/business_details_version.py#L222-L238) as it's incorrect.

Reply:
I tested with `FM1064603`.
But, I cannot see any revision for `FM1064603` in the `Alternate_names_history` table
<img width="1142" alt="Screenshot 2024-04-10 at 3 42 37 AM" src="https://github.com/bcgov/lear/assets/54437031/6b1855e6-3961-4b20-8faa-afd3d180b9be">

Reply:
so whic output are you testing?

Reply:
Sorry, it should be removed. I forgot it

Reply:
Can you make this change.  I think it will fix your problem with retrieving a business_revision

![image](https://github.com/bcgov/lear/assets/6685223/fcd45177-df34-474c-86ea-c6ce5ecdc9a3)

Reply:
Yes, I will do